oneview/data_source_server_profile.go

```diff
@@ -742,6 +742,119 @@ func dataSourceServerProfile() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+			"category": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created": {
```

Comment:
requestedMbps iis mssing and have to fixed in connection settings

Reply:
removed that line.